src/App.vue

```diff
@@ -441,10 +444,10 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     ).then(() => {
       // if we get here, Yes was clicked
       // nothing to do
-    }).catch(() => {
-      // if we get here, Cancel was clicked
-      // ignore changes
-      this.setHaveChanges(false)
+    }).catch(async () => {
+      // Delete the draft filing
+      this.getFilingId && await this.deleteFilingById(this.getFilingId)
```

Comment:
What if they previously had a draft, and they just want to cancel the current changes?

Reply:
I'll follow up with Scott, i believe the UX pin described the Cancel, as clearing the filing. Will confirm. 
If the cancel is just to return to the dashboard, it becomes simpler, no DELETE request is required.

Reply:
Yep, can confirm in the UXpin, it does say to Cancel the Alteration filing.

Reply:
So no draft alteration?

Reply:
Yea, `Resume Later` saves the draft and returns to the dashboard.

Reply:
The user can Under / Edit any changes within the filing, maybe that is given path for undoing changes, opposed to saving changes per session.

Reply:
The problem I see is if the user saves a draft, then resumes it, then makes a change, then decides to abort that change... then it deletes their saved draft!!!

Reply:
OK for now.
(As discussed, Scott will review and think about it.)